ociinstaller/plugin.go

```diff
@@ -126,18 +128,60 @@ func installPluginConfigFiles(image *SteampipeImage, tempdir string) error {
 	for _, obj := range objects {
 		sourceFile := filepath.Join(sourcePath, obj.Name())
 		destFile := filepath.Join(installTo, obj.Name())
+		if err := copyConfigFileUnlessExists(sourceFile, destFile, image.ImageRef); err != nil {
+			return fmt.Errorf("could not copy config file from %s to %s", sourceFile, destFile)
+		}
+	}
 
-		if err := copyFileUnlessExists(sourceFile, destFile); err != nil {
-			return fmt.Errorf("could not copy %s to %s", sourceFile, destFile)
+	return nil
+}
 
-		}
+func copyConfigFileUnlessExists(sourceFile string, destFile string, ref *SteampipeImageRef) error {
+	if fileExists(destFile) {
+		return nil
+	}
+	inputData, err := os.ReadFile(sourceFile)
+	if err != nil {
+		return fmt.Errorf("couldn't open source file: %s", err)
+	}
+	inputStat, err := os.Stat(sourceFile)
+	if err != nil {
+		return fmt.Errorf("couldn't read source file permissions: %s", err)
 	}
+	// transform
+	transformedData := transform(inputData, ref)
 
+	if err = os.WriteFile(destFile, transformedData, inputStat.Mode()); err != nil {
+		return fmt.Errorf("writing to output file failed: %s", err)
+	}
 	return nil
 }
 
-func pluginInstallDir(imageRef string) string {
-	ref := NewSteampipeImageRef(imageRef)
+func transform(src []byte, ref *SteampipeImageRef) []byte {
+	_, _, stream := ref.GetOrgNameAndStream()
+	if stream == "latest" {
```

Comment:
move this check to outside the function - then the renamed function name will make sense